src/lib/storage/dictionary_segment.hpp

```diff
@@ -5,13 +5,15 @@
 #include <string>
 #include <utility>
 #include <vector>
+#include <set>
```

Comment:
I guess this should be alphabetically sorted? Check back with output of lint & format scripts